pom.xml

```diff
@@ -146,13 +148,25 @@
         <version>${microprofile-config.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+          <groupId>org.eclipse.microprofile.metrics</groupId>
+          <artifactId>microprofile-metrics-api</artifactId>
+          <version>${microprofile-metrics-api.version}</version>
+          <scope>provided</scope>
+      </dependency>
 
       <dependency>
         <groupId>io.smallrye</groupId>
         <artifactId>smallrye-config</artifactId>
         <version>${smallrye-config.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-metrics</artifactId>
+        <version>${smallrye-metrics.version}</version>
+      </dependency>
```

Comment:
should it be in "test" scope?

Reply:
Err, possibly?

Currently I'd followed the same pattern as for `smallrye-config` where it's declared without a scope in the dependency management block but put into "test" scope by projects which use it as a test dependency.

For config, it looks like your quickstarts include it as a compile dependency but your modules include it as a test dependency. I'd assume that metrics would be used in a similar way.

Reply:
Yes, that should be ok, we just need to be sure it's in the `test` scope in the implementation module.

Reply:
Cool, I've now added tests and I've used `smallrye-metrics` as a test dependency.